pkg/systemd/services/timer-dialog.jsx

```diff
@@ -273,11 +278,30 @@ const CreateTimerDialogBody = ({ owner }) => {
                                 helperTextInvalid = _("Minute needs to be a number between 0-59");
                             }
 
+                            const sec = repeatPatterns[idx].second;
+                            const validationFailedSecond = !(/^[0-9]+$/.test(sec) && sec <= 59 && sec >= 0);
+
+                            if (submitted && repeat == 'minutely' && validationFailedSecond) {
+                                validated = "error";
+                                helperTextInvalid = _("Second needs to be a number between 0-59");
+                            }
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18072-20221220-142732-4b78ccb7-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)